capacity4more/modules/c4m/user/c4m_user/includes/c4m_user.theme.inc

```diff
@@ -233,7 +233,7 @@ function theme_c4m_user_country(array $field) {
   if (empty($country)) {
     return NULL;
   }
-  if ('- choose country or region -' !== drupal_strtolower($country)) {
```

Comment:
Use `t('- choose country or region -')` to avoid breaking functionality when the interface becomes multilingual…